Game.java

```diff
@@ -64,29 +59,15 @@ public void assignPlayers() {
 
     public void playGame(Board board) {
         assignPlayers();
-        print(gameMode.toUpperCase() + " game is starting..");
-        print("Where would you like to go? (1, 2, 3, 4, 5, 6, 7, 8, 9)");
-        while (!board.checkForDraw() || !board.getState().equals("Game Over")) {
+        consoleGame.drawNewGame();
+        while (!board.isGameOver()){
             board = playerMakesMove();
         }
-        System.out.println(playerActive + " has won this game!");
-    }
-
-    public enum Gamemodes {
-        PVP,
-        PVC,
-        CVC
-    }
-
-    public void displayAllGamemodes() {
-        print("Hello, what gamemode would you like?");
-        for (Gamemodes gm : Gamemodes.values()) {
-            print(gm.toString());
-        }
+        consoleGame.foundWinner();
```

Comment:
what if the game is a draw?  
foundWinner sounds like a method returning a boolean, rather than a method concerned with display.